.github/CODEOWNERS

```diff
@@ -0,0 +1,9 @@
+# Users listed in the CODEOWNERS file must review (and approve) PRs before merge.
+
+# Global code owners.
+* @ben
+
+# GitHub recommends listing the code owner in the CODEOWNER file.
+# This means changes to the CODEOWNERS file must be reviewed by the code owner.
+
+/.github/CODEOWNERS @ben
```

Comment:
Maybe you should add a "backup codeowner", so the organization can unblock itself when you're not around?